sunpy/database/database.py

```diff
@@ -819,6 +819,23 @@ def download_from_vso_query_result(self, query_result, client=None,
         self.add_many(self._download_and_collect_entries(
             query_result, client=client, path=path, progress=progress, overwrite=overwrite))
 
+    def download_from_hek_query_result(self,query_result, path=None,
```

Comment:
```suggestion
    def download_from_hek_query_result(self, query_result, path=None,
```

Reply:
Hi @dpshelio , made the changes for code consistency and pushed them. I'll remember the consistencies as I am working on another issue right now.